components/org.wso2.bfsi.identity.extensions/src/test/java/org/wso2/bfsi/identity/extensions/validator/resources/ValidatorTestDataProvider.java

```diff
@@ -0,0 +1,157 @@
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid profile bank:accounts.basic:read bank:accounts.detail:read bank:transactions:read")}
+        };
+    }
+
+    @DataProvider(name = "dp-checkValidSingleScopes")
+    public Object[][] dpCheckValidSingleScopes() {
+
+        return new Object[][]{
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.detail:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:transactions:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:payees:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:regular_payments:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid common:customer.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid common:customer.detail:read")},
+
+                //
+                {claimsTemplate.replace("${SCOPE}",
+                        "accounts openid")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid payments")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid fundsconfirmations")},
+
+                //
+                {claimsTemplate.replace("${SCOPE}",
+                        "ais openid")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "pis openid")}
+        };
+    }
+
+    @DataProvider(name = "dp-checkInValidScopeFormat")
+    public Object[][] dpCheckInValidScopeFormat() {
+
+        return new Object[][]{
+                {claimsTemplate.replace("${SCOPE}",
+                        "bank:accounts.basic:read")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.basic")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid xyz")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "xyz")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid bank:accounts.basic")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid common:customer.detail:read")},
+
+                //
+                {claimsTemplate.replace("${SCOPE}",
+                        "Accounts openid")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid Payments")},
+                {claimsTemplate.replace("${SCOPE}",
+                        "openid FundsConfirmations")},
+
+                //
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/13/commits/cd47ead8b0ab89e7914625db1aa8437e57dc4655